python/foglamp/plugins/south/systeminfo/systeminfo.py

```diff
@@ -218,13 +216,9 @@ def insert_reading(asset, time_stamp, data):
     try:
         time_stamp = utils.local_timestamp()
         get_system_info(time_stamp)
-    except asyncio.CancelledError :
-        pass
```

Comment:
why this has been removed? we should keep it.

Reply:
asyncio is no longer in use as this plugin has now become a poll plugin, handled via South C server.